api/src/pcapi/admin/custom_views/venue_view.py

```diff
@@ -405,6 +406,18 @@ def index(self):  # type: ignore [no-untyped-def]
 
         return self.index_view()
 
+    def on_model_change(self, form: Form, venue: Venue, is_created: bool = False) -> None:
+        previous_value = form._fields["isPermanent"].object_data
+        new_value = venue.isPermanent
+        if new_value and previous_value != new_value:
+            send_permanent_venue_needs_picture(
+                [venue.bookingEmail]
+                if venue.bookingEmail
+                else [recipient.user.email for recipient in venue.managingOfferer.UserOfferers],
```

Comment:
Vérifie qu'il faut effectivement l'envoyer à tous les utilisateurs rattachés ou ayant demandé un rattachement.
J'imagine qu'en fait il ne faudrait que les utilisateurs dont le rattachement est validé : 
`[user_offerer.user.email for user_offerer in venue.managingOfferer.UserOfferers if user_offerer.isValidated]`

Reply:
Je vérifie ça. Par contre, il semble y avoir un décalage front/back avec ce champ qui est obligatoire côté front.

Reply:
Parles-tu du champ `bookingEmail` obligatoire côté front ?
Même s'il l'est pour les nouvelles validations de formulaire, en l'état actuel sur staging (et donc sans doute en prod) on a beaucoup de `bookingEmail` vides sur les lieux existants : 
```
pcapi-staging=> select count(*) from venue where "bookingEmail" is null or "bookingEmail" = '';
 count 
-------
 21861
(1 row)
```

Reply:
Oui, j'ai vu ça. J'ai remonté l'info, j'attends validation de quel mail il faut utiliser.

Reply:
Au final, le mail ne sera envoyé qu'au `bookingEmail`. S'il n'existe pas, le mail n'est pas envoyé.